src/components/vocabulary/Vocab.js

```diff
@@ -111,6 +115,11 @@ const vocabulary = {
     description: "Class for a property template",
     url: "http://sinopia.io/vocabulary/PropertyTemplate",
   },
+  "literalPropertyAttribute/userIdDefault": {
```

Comment:
should this start with "has" ?  "hasLiteralAttributeUserIdDefault" ?

Reply:
This is referencing the `userIdDefault` in `literalPropertyAttribute` so this is consistent naming.